docker-compose.yaml

```diff
@@ -38,6 +38,7 @@ services:
       , "--peer", "hydra-node-2:5001"
       , "--peer", "hydra-node-3:5001"
       , "--hydra-scripts-tx-id", "${HYDRA_SCRIPTS_TX_ID}"
+      , "--contestation-period", "1"
```

Comment:
Only 1 second for contestation?

Reply:
This is used for E2E tests, so that way it works fast.